tests/integration/test_integ.py

```diff
@@ -0,0 +1,129 @@
+# HOW TO EXECUTE?
+# First install the package: pip install pytest-docker-compose
+# Run with: pytest --docker-compose="path_to_docker_compose_yml" test_integ.py
```

Comment:
why having an extra argument for pytest? the docker-compose file should always be the same place and if it changes, you adapt the path to it. there is btw. a nice library called `yaml` with which you can read out the docker-compose file and getting the container name: https://stackoverflow.com/questions/1773805/how-can-i-parse-a-yaml-file-in-python

Reply:
I just added  -f argument to docker-compose up pointing it to the location of the docker file.